packages/odyssey-react-mui/src/Radio.tsx

```diff
@@ -114,17 +129,30 @@ const Radio = ({
         <MuiRadio
           inputProps={{
             "data-se": testId,
+            "aria-disabled": isReadOnly,
+            readOnly: isReadOnly,
+            tabIndex: isReadOnly ? 0 : undefined,
           }}
           inputRef={localInputRef}
           onChange={onChange}
+          onClick={onClick || handleClick}
+          disabled={isDisabled}
         />
       }
-      disabled={isDisabled}
       label={label}
       name={name}
       translate={translate}
       value={value}
       onBlur={onBlur}
+      disabled={isDisabled}
+      sx={{
+        ...(isReadOnly && {
+          cursor: "default",
+          "& .MuiTypography-root": {
+            cursor: "default",
+          },
+        }),
+      }}
```

Comment:
Usually, these styles are in `components.tsx` so they apply to all inputs. Can we move them in there?

Reply:
This style only applies to the label, not the form elements. I wanted this to live in `components.tsx` but found I was not able to target from the MuiCheckbox/MuiRadio style blocks, since its applied outside of the component. I believe we'd have to re-factor how the `MuiFormControlLabel` works to accept the isReadOnly prop so I figure this is the easier and cleaner path since it only applies to checkboxes and radios.